documentation/book/security.adoc

```diff
@@ -1,27 +1,37 @@
 == Security
 
-The Apache Kafka project supports data encryption by means of the SSL/TLS protocol.
+The Strimzi project supports data encryption of communication between the different Kafka and Strimzi components by means of the SSL/TLS protocol.
 This makes it possible to encrypt data transferred between brokers (interbroker communication) and between clients and brokers.
+For the internal components, TLS certificates are also used for authentication.
```

Comment:
It's not really clear to a general reader what the _internal components_ are

Reply:
Fair enough. I know use either the term _Kafka and Strimzi components_ or just _components_ and list them on the beginning (Kafka brokers, Zookeeper nodes, Kafka Connect, Strimzi Topic Operator).